src/metadata/metadata_handler.h

```diff
@@ -176,6 +177,7 @@ static constexpr auto res_keys = std::array<std::pair<resource_attributes_t, con
     { R_FANART_OBJ_ID, "fanArtObject" },
     { R_FANART_RES_ID, "fanArtResource" },
     { R_BITS_PER_SAMPLE, "bitsPerSample" },
+    { R_LANGUAGE, "dc:language" },
```

Comment:
array size is not updated.

Reply:
`R_MAX` is increased by adding the new enum entry.

Reply:
Ah OK

Reply:
Thanks for the hint: the entry is still missing in `script_names.h`